apis/stacks/v1alpha1/types.go

```diff
@@ -465,7 +465,9 @@ type ContributorSpec struct {
 // ControllerSpec defines the controller that implements the logic for a stack,
 // which can come in different flavors.
 type ControllerSpec struct {
-	StackControllerOptions `json:",inline"`
+	// ServiceAccount options allow for changes to the ServiceAccount
+	// the Stack Manager creates for the Stack's controller
+	ServiceAccount *ServiceAccountOptions `json:"serviceAccount,omitempty"`
```

Comment:
I don't think the field should be named as `serviceAccount` when it's actually `ServiceAccountOptions`, not actual `ServiceAccount`.

Reply:
@muvaf This is post merge but I still wanted to respond. I agree that the name "serviceAccount" may suggest that we have a complete service account object beneath this field, but counter examples are "Stack.Controller.Deployment" which is a DeploymentSpec.  There are other examples of a `*Spec` type being used with the field label omitting "spec" from the name.

Reply:
Understood, though I think we should change that one, too, in the future.